src/v1t/models/readout/gaussian2d.py

```diff
@@ -237,6 +237,7 @@ def sample_grid(self, batch_size: t.Union[int, torch.Tensor], sample: bool = Non
     def forward(
         self,
         inputs: torch.Tensor,
+        query_neurons: torch.Tensor = None, 
```

Comment:
as the default value is None - it should be 
```suggestion
        query_neurons: Optional[torch.Tensor] = None, 
```
It should also have `from typing import Optional` in the beginning

Reply:
is `query_neurons` actually used anywhere here? have I missed it?